The Sequence Read Archive (SRA) stores sequencing data from the next generation of sequencing platforms including:

Roche 454 GS System;

Illumina Genome Analyzer;

Life Technologies AB SOLiD System;

Helicos Biosciences Heliscope;

Complete Genomics;

Pacific Biosciences SMRT;

If mutation in gene need transplantation of stem cells of healthy donor

Various methods are available for detecting and quantitating gene expression levels, including northern blots6, S1 nuclease pro- tection7, differential display8, sequencing of cDNA libraries9,10 and serial analysis of gene expression11 (SAGE). Augmenting this coterie are two array-based technologies — cDNA and oligonu- cleotide arrays.

This is illustrated by diverse applications that include: investigating gene expression in the roots and leaves of Arabidopsis thaliana3, human T cells exposed to phorbol ester12, rheumatoid arthritis and inflammatory bowel disease16, tumori- genic versus non-tumorigenic cell lines4, the diauxic shift from anaerobic to aerobic metabolism in S. cerevisiae5,17 (yeast)murine T cells challenged with 4-phorbol-12-myristate-13- acetate13 and in Streptococcus pneumoniae18

reverse transcription from an oligo-dT primer is used

Genetics vs Genomics

Genetics study of heredity.

Genomics study of how genes interact and expressed s a whole.

Genomics and gene expression profiling tools focus on the cancer itself

and help to determine:

- How aggressive is the cancer(prognosis)

- What likely benefit from treatment(prediction)

Genetics tests finds mutations not desease

Expression profiling using cDNA microarrays

cDNA microarrays are capable of profiling gene expression patterns of tens of thousands of genes in a single experiment

Researchers have catalogued more than 1.1 million expressed sequence tagged sites (ESTs), corresponding with 52,907 unique human genes

expressed sequence tags (ESTs)

expressed sequence tagged sites (ESTs)

Two types of microarray

- Slide

- Chip

Types of gene expression assays(испытание):

- Serial Analysis of gene expression(SAGE)

- Short oligonucleotide arrays(affymetrix) DNA CHIP

- Long oligonucleotide arrays(Agilent inkjet)

- Fibre optic arrays(Illumnia)

- cDNA arrays(Brown/Botstein)

generated Signal depends the strength of hybridization determined by number of paired bases

The ability to identify genes at the nucleic acid level rather than proceeding from a known protein to its chromosomal counter- part has prompted efforts to likewise extract functional informa- tion at the nucleic acid level

exploring the function of a gene

detecting and quantitating gene expression levels,

Various methods are available for detecting and quantitating gene expression levels, including northern blots6, S1 nuclease pro- tection7, differential display8, sequencing of cDNA libraries9,10 and serial analysis of gene expression11 (SAGE). Augmenting this coterie are two array-based technologies — cDNA and oligonu- cleotide arrays. These allow one to study expression levels in par- allel3,12,13, thus providing static information about gene expression (that is, in which tissue(s) the gene is expressed) and dynamic information (that is, how the expression pattern of one gene relates to those of others). The high degree of digital data extraction and processing of these techniques supports a variety of samples or experimental conditions.

cDNA and oligonucleotide arrays are capable of analysing patterns of gene expression, fundamental differences exist between the methods.

Microarray small silica or glass slide can contain 60.000 or more different gene sequences(15-25 bp) attached to glass side. Oligonucleotide sequences represents uniq sequences of different genes of genom

Each spot contains piece of DNA with a known sequence

Production of microarray:

Glass slide 1.5x1.5 cm

We cover glass side by template witch contains holes in specific positions

Place one type of nucleotide acid(A or T or C or G)

Light activation reaction

Then we repeat this process applying another template.

At the end we will generate glass slide with different sequences of oligonucleotides.

Microarray doesn't have any holes!! Oligonucleotide sequences binds to the surface of microarray

Normal tissue

Tumor tissue

Both tissues produce mRNA

Obtain mRNA from tissue

Obtain cDNA from mRNA

Make fluorescent Labeling molecules of cDNA:

normal tissue - green fluorescent

Tumor tissue - red fluorescent

Mix cDNA samples

microarray has nucleotide sequences probes before adding cDNA

Add mixed sample to microarray

fluorescent reaction occurs if nucleotide sequences in probes binds to added sequences

cDNA Hybridization step

Washing array, to get rid of unattached sequences.

laser scanner

Laser activare fluorescent

In the array well a lot of complimentary mRNA strands which compliment to specific gene

mRNA reverse transcribe to more stable cDNA

If Gene A upregulated in tumor tissue we will see strong red signal and weak green signal

If Gene B downregulated in tumor tissue we will see strong weak signal and strong green signal.

If Gene C didn't changed we will see combination of red and green signals.

or down regulated

Incase of cancer we can have different patterns of signals. We can define which pattern associated with good or bad prognosis. For example we can avoid chemotherapy in case if we have certain patterns which associated with good prognosis. Or if we have bad prognosis cancer we have to apply more aggressive treatment.