|   |   |   |
|---|---|---|
|1|AGO-IP<br><br>Degradome||
|2|pSILAC||
|3|qPCR||
|4|RIP-seq|RIP - RNA binding protein immunoprecipitation|
|5|RNA-seq||
|6|RPF-seq||
|7|SILAC (stable isotope labeling with amino acids in cell culture)||
|8|TRAP||
|9|Western Blotting|For Protein detection, also for detection chemical reactions in the cell<br><br>1. Break cellular cell and separate proteins from organels, cell wall by using centrifuge<br>2. Protein mix, separation of proteins based on charge, size/shape, mass. Use electrophoresis. SDS-Page<br>3. Probing|
||SDS-PAGE (sodium dodecyl sulfate-poly- acrylamide gel electrophoresis)|provides 1-dimensional (1D) results when analyzing proteins by size.<br><br>If the target protein is too small for the band (1D) or spot (2D) to be easily stained after electrophoresis, a Western blot using antibodies can be performed. In order to perform Western blot- ting, it is necessary to have an antibody specific to the target protein.|
|10|Northern Blotting|For RNA detection|
||Southern Blotting|For DNA detection|
|11|Flow Cytometry||
|12|Genetic testing||
|29|PAR-CLIP|validation of miRNA target sites.<br><br>Photo- activatable-Ribonucleoside-Enhanced Crosslinking and Immunoprecipitation<br><br>CLIP method was modified in respect of an ultraviolet irradiation used to covalently crosslink RNA-protein complexes within cells<br><br>It frequently encounter problems with distinguishing between direct and indirect miRNA-mRNA interactions.<br><br>[PAR-CLIP](https://en.wikipedia.org/wiki/PAR-CLIP) (photoactivatable ribonucleoside–enhanced cross-linking and immunoprecipitation) is a biochemical method used for identifying the binding sites of cellular RNA-binding proteins (RBPs) and microRNA-containing ribonucleoprotein complexes (miRNPs).[[25]](https://en.wikipedia.org/wiki/Cross-linking_immunoprecipitation#cite_note-FOOTNOTEHafnerLandthalerBurgerKhorshid2010-25) The method relies on the incorporation of photoreactive ribonucleoside analogs, such as 4-thiouridine (4-SU) and 6-thioguanosine (6-SG) into nascent RNA transcripts by living cells. Irradiation of the cells by UV light of 365 nm induces efficient cross-linking of photoreactive nucleoside-labeled cellular RNAs to interacting RBPs. Immunoprecipitation of the RBP of interest is followed by the isolation of the cross-linked and co-immunoprecipitated RNA. The isolated RNA is converted into a cDNA library and deep sequenced using [high-throughput sequencing](https://en.wikipedia.org/wiki/High-throughput_sequencing) technology.[[25]](https://en.wikipedia.org/wiki/Cross-linking_immunoprecipitation#cite_note-FOOTNOTEHafnerLandthalerBurgerKhorshid2010-25)[[28]](https://en.wikipedia.org/wiki/Cross-linking_immunoprecipitation#cite_note-FOOTNOTEHafnerLandthalerBurgerKhorshid2010b-28) Cross-linking the 4-SU and 6-SG analogs results in thymidine to cytidine, and guanosine to adenosine transitions respectively. As a result, PAR-CLIP can identify binding site locations with high accuracy.[[4]](https://en.wikipedia.org/wiki/Cross-linking_immunoprecipitation#cite_note-FOOTNOTEZhangDarnell2011-4)<br><br>However, PAR-CLIP is limited to cultured cells,[[4]](https://en.wikipedia.org/wiki/Cross-linking_immunoprecipitation#cite_note-FOOTNOTEZhangDarnell2011-4)[[27]](http://) and nucleoside cytotoxicity is a concern;[[3]](http://)[[25]](https://en.wikipedia.org/wiki/Cross-linking_immunoprecipitation#cite_note-FOOTNOTEHafnerLandthalerBurgerKhorshid2010-25) it has been reported that 4-SU inhibits ribosomal RNA synthesis, induces a nucleolar stress response, and reduces cell proliferation.[[29]](http://) 4-SU substitution occurs in approximately 1 out of every 40 uridine nucleosides, and that T to C transitions frequently occur at the cross-link site.[[25]](https://en.wikipedia.org/wiki/Cross-linking_immunoprecipitation#cite_note-FOOTNOTEHafnerLandthalerBurgerKhorshid2010-25)<br><br>Recently, PAR-CLIP has been employed to determine the transcriptome-wide binding sites of several known RBPs and microRNA-containing ribonucleoprotein complexes at high resolution. This includes the miRNA targeting AGO and TNRC6 proteins.[[22]](https://en.wikipedia.org/wiki/Cross-linking_immunoprecipitation#cite_note-FOOTNOTEYangLiShaoZhou2011-22)[[25]](https://en.wikipedia.org/wiki/Cross-linking_immunoprecipitation#cite_note-FOOTNOTEHafnerLandthalerBurgerKhorshid2010-25)|
|13|HITS-CLIP  <br>(CLIP-Seq)|CLIP - Cross-linking and ImmunoPrecipitation; High-throughput sequencing of RNA isolated by crosslinking immunoprecipitation.<br><br>HITS-CLIP of the RNA-binding protein [Argonaute](https://en.wikipedia.org/wiki/Argonaute) has been performed for the identification of microRNA targets.<br><br>The process of using antibody antigen interaction to separate proteins from mixture of proteins.<br><br>Insoluble parts which helps to drag the antibody with protein down.<br><br>Precipitants - осадители. Beads - Agorose<br><br>Co-ImmunoPrecipitation - precipitation of large protein complex<br><br>Upon UV exposure, covalent bonds are formed between proteins and nucleic acids that are in close proximity<br><br>[HITS-CLIP](https://en.wikipedia.org/wiki/HITS-CLIP),[[20]](https://en.wikipedia.org/wiki/Cross-linking_immunoprecipitation#cite_note-FOOTNOTEDarnell2010-20) also known as CLIP-Seq, combines UV [cross-linking](https://en.wikipedia.org/wiki/Cross-link) and [immunoprecipitation](https://en.wikipedia.org/wiki/Immunoprecipitation) with [high-throughput sequencing](https://en.wikipedia.org/wiki/High-throughput_sequencing) to identify binding sites of RNA-binding proteins. CLIP-seq depends on cross-linking induced mutation sites (CIMS) to localized protein-RNA binding sites.[[4]](https://en.wikipedia.org/wiki/Cross-linking_immunoprecipitation#cite_note-FOOTNOTEZhangDarnell2011-4) Because CIMS are reproducible, [high sequencing depths](https://en.wikipedia.org/wiki/Deep_sequencing) allow CIMS to be differentiated from technical errors.|
||TAGS|GST;GFP;poly hist|
|14|iCLIP|[iCLIP](https://en.wikipedia.org/wiki/ICLIP) (individual nucleotide–resolution cross-linking and immunoprecipitation) is a technique used for identifying protein-RNA interactions. The method uses [UV light](https://en.wikipedia.org/wiki/UV_light) to covalently bind proteins and RNA molecules. As with all CLIP methods, iCLIP allows for the stringent purification of linked protein-RNA complexes, using immunoprecipitation followed by [SDS-PAGE](https://en.wikipedia.org/wiki/SDS-PAGE) and membrane transfer. The radiolabelled protein-RNA complexes are then excised from the membrane, and treated with proteinase to release the RNA. This leaves one or two amino acids at the RNA cross-link site. The RNA is then reverse transcribed using barcoded primers. Because reverse transcription stops prematurely at the cross-link site, iCLIP allows RNA-protein interaction sites to be identified at high resolution. As a novel modification of iCLIP, m6A-CLIP further took advantage of the m6A-induced truncation sites (MITS) to precisely map m6A sites in mRNAs.[[5]](https://en.wikipedia.org/wiki/Cross-linking_immunoprecipitation#cite_note-FOOTNOTEKeAlemuMertensGantman2015-5)|
||CHIP-seq|Chromatin Immunoprecipitation combined with high-throughput sequencing.<br><br>DNA protein interactions. For example need to identify a target of novel Transcript factor.<br><br>It identifies the locations of genome bounds by proteins<br><br>1. Separate DNA regions attached to antibody<br>2. Do sequence|
|15|Immunofluorescence||
|16|Immunohistochemistry||
|17|IMPACT-seq||
|18|In situ hybridization||
|19|LAMP Loop-mediated isothermal amplification||
|20|Luciferase Reporter Assay|validation of miRNA target sites.<br><br>This method is based on cloning 3’UTRs of genes of interest or 3’UTR segments containing miRNA binding site(s) into expression vectors bearing a reporter gene. As a negative control, constructs carrying 3’UTRs with mutated target sites that unable miRNAs binding are also used. Transient transfection of cells with reporter constructs followed by measurement of the reporter activity enables validation of predicted miRNA-mRNA interactions. Cells not expressing a miRNA of interest may be co-transfected with the reporter construct and either a miRNA mimetic or a miRNA-encoding vector. This method is frequently extended and the miRNA-mRNA interaction is further confirmed by a transfection with miRNA inhibitors. The reporter assay still serves as an efficient and routinely used strategy for the verification of individual miRNA- mRNA interactions. not allow a high-throughput identification of miRNA targets is an important drawback.<br><br>more sensitive methods (e.g., reporter assays) cannot be used in large-scale analyses.<br><br>Certain targets validated by reporter gene assays may be also false negatives as an effect of RISC saturation due to miRNA overexpression.|
|21|Microarrays analysis|validation of miRNA target sites.<br><br>high-throughput method used in the experimental validation of miRNA- mRNA interactions is microarray analysis.<br><br>Such an analysis consists in the comparison of cell transcriptomes after miRNA overexpression or knockdown with reference to the transcriptome of untreated cells.<br><br>The significant advantage of this method ids the fact that it enables performing a large-scale analysis. However, not all miRNA-mRNA interactions may be discovered by this method since miRNAs can downregulate genes by lowering protein expression levels with an undetected change in transcript levels. Moreover, the transcriptomes vary between different tissues and highly depend on the cell physiology. Therefore, a strategy involving multiple set of microarrays was proposed to overcome these problems and ensure reliability of the method.|
|22|Proteome analysis|validation of miRNA target sites.<br><br>proteomics approach is based on measuring the change of protein level in response to miRNA introduction.<br><br>This method employs stable isotope labeling with amino acids in cell culture (SILAC) followed by a quantitative-mass-spectrometry.<br><br>Later, this technique was modified by conducting pulse labeling with two different heavy isotopes|
|23|5'RACE|validation of miRNA target sites.<br><br>Furthermore, methods based on different approaches were also successfully implemented for the verification of miRNA-mRNA interactions. Davis et al. [92] conducted RNA-ligase-mediated (RLM) 5’ RACE experiments to verify miRNA-target interactions, and Li et al. [93] proposed that high cytoplasmic-to-nucleic ratio of mRNA expression may indicate genes that are likely to be controlled by miRNAs. In the latter report it was suggested that a measurement of the transcript level by a microarray or fluorescent real time PCR (qPCR) preceded by a computational analysis can be performed to create a list of potential targets|
|24|immunoblotting|validation of miRNA target sites.|
|25|mRNA-RISC<br><br>coimmunoprecipitatiopn|validation of miRNA target sites.<br><br>AGO proteins of RISC can bind both miRNAs and mRNAs and this feature was exploited in co-immunoprecipitation assays.<br><br>Used collectively with deep sequencing, AGO immunoprecipitation allowed validating of miRNA- mRNA interactions in a genome-wide manner. HITS- CLIP, namely high-throughput sequencing of RNAs isolated by crosslinking immunoprecipitation, was used to directly identify AGO-bound miRNAs and their target mRNAs in the mouse brain.|
|26|Electrophoretic Mobility Shift Assay||
|27|2D-DIGE||
|28|3LIFE||
|30|PARE||
|31|Biotin-microarrays||
|32|Biotin-qPCR||
|33|Biotin-seq||
|34|CLASH|validation of miRNA target sites.|
|35|CLEAR-CLIP||
|36|Degradome||
|37|ELISA||